src/contracts/Model/Suggestion/Suggestion.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Search\Model\Suggestion;
+
+use Ibexa\Contracts\Core\Repository\Values\ValueObject;
+
+abstract class Suggestion extends ValueObject
+{
+    private float $score;
+
+    private string $name;
+
+    private string $pathString;
+
+    private ParentLocationCollection $parentsLocation;
+
+    /**
+     * @param array<\Ibexa\Contracts\Search\Model\Suggestion\ParentLocation> $parentLocations
+     */
+    public function __construct(
+        float $score,
+        string $name,
+        string $pathString = '',
+        array $parentLocations = []
+    ) {
+        $this->score = $score;
+        $this->name = $name;
+        $this->pathString = $pathString;
+        $this->parentsLocation = new ParentLocationCollection($parentLocations);
+
+        parent::__construct();
+    }
+
+    public function getScore(): float
+    {
+        return $this->score;
+    }
+
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+    public function getPathString(): string
+    {
+        return $this->pathString;
+    }
+
+    public function getParentLocations(): ParentLocationCollection
```

Comment:
`Suggestion` are meant to be generic to accommodate different object types in the future however they still use `ParentLocation` objects tightly specific to Content items. I think you should remove ParentLocations from `Suggestion` entirely and leave it as a part of `ContentSuggestion` only. I suppose in the future we are going to suggest things like Content Types, Content Type Groups, Sections etc. - they are not hierarchical and we probably won't need to display paths for them.